src/java.base/windows/classes/sun/nio/fs/WindowsFileStore.java

```diff
@@ -232,12 +241,20 @@ public boolean equals(Object ob) {
         if (!(ob instanceof WindowsFileStore))
```

Comment:
We can change this to
if (ob instanced WindowsFileStore other) { ... } and it will avoid the cast.